sdk/identity/identity/src/credentials/visualStudioCodeCredential.ts

```diff
@@ -146,15 +131,89 @@ export class VisualStudioCodeCredential implements TokenCredential {
   }
 
   /**
-   * Runs preparations for any further getToken request.
+   * Load the broker plugin from @azure/identity-broker package.
+   */
+  private async loadBrokerPlugin() {
+    try {
+      // @ts-ignore
```

Comment:
maybe remove the ts-ignore and see what the actual error is. That will help understand why vi-test is yelling?